### PR TITLE
Scala 2.13.7 release notes

### PR DESCRIPTION
**Pull request by SethTisue**
feedback on notes is welcome — comment here

### CODE REVIEW

File: releases/2.13.7.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The [release notes for Scala 2.13.0](https://github.com/scala/scala/releases/v2.13.0) have important information applicable to the whole 2.13 series.
+
+## Obtaining Scala
+
+Scala releases are available through a variety of channels, including (but not limited to):
+
+* Bump the `scalaVersion` setting in your sbt-based project
+* Download a distribution from [scala-lang.org](http://scala-lang.org/download/2.13.6.html)
+* Obtain JARs via [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.scala-lang%22%20AND%20v%3A%222.13.6%22)
```
Comment (Philippus) [inline releases/2.13.7.md]:
```suggestion
* Download a distribution from [scala-lang.org](http://scala-lang.org/download/2.13.7.html)
* Obtain JARs via [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.scala-lang%22%20AND%20v%3A%222.13.7%22)
```

**Review by Philippus — CHANGES_REQUESTED** [review]

File: releases/2.13.7.md (hunk 1 above)
Comment (SethTisue) [reply releases/2.13.7.md]: good catch, thanks!

File: releases/2.13.7.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+The [release notes for Scala 2.13.0](https://github.com/scala/scala/releases/v2.13.0) have important information applicable to the whole 2.13 series.
+
+## Obtaining Scala
+
+Scala releases are available through a variety of channels, including (but not limited to):
+
+* Bump the `scalaVersion` setting in your sbt-based project
+* Download a distribution from [scala-lang.org](http://scala-lang.org/download/2.13.7.html)
+* Obtain JARs via [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.scala-lang%22%20AND%20v%3A%222.13.7%22)
```
Comment (xuwei-k) [inline releases/2.13.7.md]:
```suggestion
* Download a distribution from [scala-lang.org](https://scala-lang.org/download/2.13.7.html)
* Obtain JARs via [Maven Central](https://search.maven.org/search?q=g:org.scala-lang%20AND%20v:2.13.7)
```

- https://github.com/scala/scala-dev/pull/797

Comment (michelou) [reply releases/2.13.7.md:66]: @xuwei-k Query filters (i.e. `g:` for group and `v:` for version) must be separated by an `&` (ampersand), not by a `%20` (space).

Comment (michelou) [reply releases/2.13.7.md]: @xuwei-k Query filters (i.e. **`g:`** for group and **`v:`** for version) must be separated by **`%20AND%20`** and not just **`%20`** (space). Tested with Firefox.

Comment (xuwei-k) [reply releases/2.13.7.md]: Thanks. updated

Comment (SethTisue) [reply releases/2.13.7.md]: thank you

**Comment by SethTisue** [comment]
The notes are now at https://github.com/scala/scala/releases/tag/v2.13.7 . Further edits remain possible; I'll continue monitoring comments/suggestions here.

File: releases/2.13.7.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+### Collections
+
+* Make `ArrayBuffer`'s iterator fail fast when buffer is mutated ([#9258](https://github.com/scala/scala/pull/9258) by [@NthPortal](https://github.com/NthPortal))
+* Fix `ArrayOps` bugs (by avoiding `ArraySeq#array`, which does not guarantee element type) ([#9641](https://github.com/scala/scala/pull/9641) by [@som-snytt](https://github.com/som-snytt))
+* Deprecate `IterableOps.toIterable` ([#9774](https://github.com/scala/scala/pull/9774) by [@lrytz](https://github.com/lrytz))
+
+### Other changes
+
+* Accept supplementary Unicode characters in identifiers ([#9687](https://github.com/scala/scala/pull/9687) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Vimplicits-max-modules` to control how types are printed in `-Vimplicits` messages ([#9767](https://github.com/scala/scala/pull/9767) by [@mrdziuban](https://github.com/mrdziuban))
```
Comment (mrdziuban) [inline releases/2.13.7.md:35]: @SethTisue this PR ended up actually changing the behavior to print full type names unless the type is in the pre-defined list of shorthands, as suggested in https://github.com/scala/scala/pull/9767#issuecomment-931125211, so there's not a new `-Vimplicits-max-modules` option. Apologies, I should have changed the name of the PR after addressing the comment

Comment (SethTisue) [reply releases/2.13.7.md:35]: @mrdziuban Thanks, good catch! I've removed this bullet from the release notes. Do please change the Scala PR title and description to match the real change

**Pull request closed without merging**